### ISSUE TITLE
Can not apply document dialect

### ISSUE THREAD

**Issue by ricardoavelino** [znck/grammarly#362]
I am writing a document with British English, and the settings have worked so far. 

However, today I noticed that even by changing the dialect to `british`, it still mark errors regarding to the `american` dialect.

Is there something that I can try to make this work properly again? I already tried to restart the server multiple times.

**Comment by rayvburn** [comment]
I can confirm that this issue has appeared recently. The old configuration seems to be valid since a warning does not appear with:

```json
"grammarly.config.documentDialect": "british",
```

**Comment by ricardoavelino** [comment]
Have you found a way around it @rayvburn ?

**Comment by rayvburn** [comment]
Nope. I've tried to delete the VSCode's workspace configuration and set `british` in the global preferences, but the text is still being checked against `american` dialect.

**Comment by adriancaruana** [comment]
I'm also having this issue, would be great for it to be fixed!

**Comment by haselwarter** [comment]
Same; did the dialect API change?

**Comment by josehernandezvargas** [comment]
I'm also having this issue despite "grammarly.config.documentDialect": "british". 
Is there a way to ignore all the warnings related to "Non-American variants"

**Comment by rayvburn** [comment]
I have a feeling that the bug appears only in `.tex` documents. Recently, I have been writing some markdown documentation and Grammarly did not complain about british words. Could you guys confirm?

I'm also using LaTeX Workshop to work with `.tex`, if that matters.

**Comment by josehernandezvargas** [comment]
I just copied the content of the .tex file into an .md file and it still complains about British spellings.
I'm also using LaTeX workshop with .tex files by the way.

**Comment by janrito** [comment]
> I have a feeling that the bug appears only in `.tex` documents. Recently, I have been writing some markdown documentation and Grammarly did not complain about british words. Could you guys confirm?
> 
> I'm also using LaTeX Workshop to work with `.tex`, if that matters.

I have this problem in markdown as well

**Comment by Samreay** [comment]
Another +1 to this issue. Id be happy to look into it, but is this extension still being maintained? I see a bunch of open PRs and I'm hesitant to dig through code if a change might sit as a PR for an indefinite amount of time.

**Comment by vzaliva** [comment]
I am also affected.

**Comment by ImLunaHey** [comment]
@znck is there a fix for this?

**Comment by junglegobs** [comment]
Australian seems to work and at least it stops telling me that "minimize" is wrong ("color" is marked as wrong though, see this article: https://www.grammarly.com/blog/how-to-switch-dialects/)

**Comment by vzaliva** [comment]
I have "British English" in Grammarly account preferences on their website. Yet when using from emacs, I am getting "organisation" highlighted, and it suggests changing to "organization". My `lsp-grammarly-dialect` is set to `"british"`.

**Comment by josehernandezvargas** [comment]
As a workaround, I just added British spellings (colour, optimise, etc.) to my personal dictionary on the Grammarly account and it stopped highlighting them.

**Comment by isaldiviagonzatti** [comment]
@znck Hi! Is there a fix to this? Is this extension being maintained? Thanks

**Comment by SidSidSid16** [comment]
I'm also having the same issue.
